gui/package.json

```diff
@@ -21,11 +21,14 @@
     "@mui/icons-material": "^5.15.17",
     "@mui/material": "^5.15.17",
     "monaco-editor": "^0.48.0",
+    "plotly.js": "^2.33.0",
```

Comment:
Thanks for pointing that out, I didn't know it was available.

Not an easy change right now since I am using react-plotly (npm package) which seems to only work with the full plotly.js. But I'm sure there's a workaround.

I'll note that since we are doing lazy loading, the plotly.js doesn't contribute to the main bundle on initial page load.

Shall we defer for a separate issue when we can figure out how to work around limitations of react-plotly?

Reply:
Sure, we can open a ticket to try it later. I know react-plotly [supports](https://github.com/plotly/react-plotly.js/blob/master/README.md#customizing-the-plotlyjs-bundle) it, but I’m not sure how this interacts with the lazy loading, and as you say the lazy loading helps make it less necessary

Reply:
For information, I just tried using a smaller distribution and ran into some bundling problems, so yeah I think we should do this in a different ticket.